app/dossiers/dossier/page.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+
+import { Layout } from "@/app/components/Layout";
+import AdditionalInfoCard from "@/components/folders/AdditionalInfoCard";
+import Hero from "@/components/folders/HeroSection";
+import ContextCard from "@/components/folders/ContextCard";
+import CommiteeCard from "@/components/folders/CommiteeCard";
+import ForcesCard from "@/components/folders/ForcesCard";
+import LegislativeDocuments from "@/components/folders/LegislativeDocuments";
+import StatusChip from "@/components/StatusChip";
+import SpeakingTimeCard from "@/components/folders/SpeakingTimeCard";
+import TextStructureCard from "@/components/folders/TextStructureCard";
+import TimelineCard from "@/components/folders/TimelineCard";
+
+import Box from "@mui/material/Box";
+
+import Stack from "@mui/material/Stack";
+import TabsSection from "@/components/folders/TabsSection";
+
+export default function Dossier() {
+  return (
+    <Layout>
+      <Hero />
+      <Box alignSelf="center" pb={2} width={1024}>
```

Comment:
I tend to prefer putting every style-related thing into a [sx prop](https://mui.com/system/getting-started/the-sx-prop/) to distinguish style and logic props. But what you did works well too

```suggestion
      <Box sx={{
        alignSelf: "center",
        pb: 2
        width: 1024
      >
```

Reply:
Agreed 👍